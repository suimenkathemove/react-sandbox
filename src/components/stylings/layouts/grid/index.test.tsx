import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { Grid } from './';

const setup = () => render(<Grid {...defaultProps} />);

describe('Grid', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
