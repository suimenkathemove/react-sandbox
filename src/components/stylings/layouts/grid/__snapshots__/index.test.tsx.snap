// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid render 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL kLkpgg"
  >
    <header
      class="sc-gsDKAQ ijA-dTo"
    />
    <main
      class="sc-dkPtRN fnGakI"
    />
    <footer
      class="sc-hKwDye dEtGwW"
    />
  </div>
</DocumentFragment>
`;
